js/script.js

```diff
@@ -5,6 +5,25 @@ const outlineButtonsLink = document.querySelectorAll( '.wp-block-button.is-style
 const clipProperties = ['--topLeftX', '--topLeftY', '--topRightX', '--topRightY', '--bottomLeftX', '--bottomLeftY', '--bottomRightX', '--bottomRightY' ];
 
 
+/************** 
+  Site Title 
+	  remove text to replace with image
+	  make the text screen reader text
+ **************/
+
+let siteTitle = document.querySelector('.site-title a');
```

Comment:
Hmm...definitely don't want to do this with JavaScript :( I added a template override for the header branding to this branch, @zoracreates can you pull that down and update this in the markup?